charger/easee.go

```diff
@@ -280,6 +281,50 @@ func (c *Easee) ProductUpdate(i json.RawMessage) {
 		// received observation is outdated, ignoring
 		return
 	}
+
+	switch res.ID {
+		case easee.CHARGER_OP_MODE:
+			//for every op mode change, if last energy updated was triggered more than 3 minutes ago, request new update
+	  		if c.lastEnergyPollTriggered.Before(time.Now().Add(-3 * time.Minute)) {
+				uri := fmt.Sprintf("%s/chargers/%s/commands/%s", easee.API, c.charger, easee.PollLifetimeEnergy)
+				c.Post(uri, request.JSONContent, request.MarshalJSON(nil))  //do not wait for success, fire&forget
+				c.lastEnergyPollTriggered = time.Now();
+				//TODO: wait for async update of LIFETIME_ENERGY?
```

Comment:
this actually won't be as easy as I thought, as we cannot wait for another ProductUpdate while already processing one. The race condition hunch of mine so far is exactly that - a hunch. Let's try and see if and how often we really hit it. Maybe the problem is purely theoretical.

Reply:
I think we can work around this problem by explicitly fetching the Lifecycle Energy via the API.
I just tested this API call here: https://developer.easee.cloud/reference/get_api-sites-siteid-energy

It gives energy data per charger like this:

```
[
  {
    "chargerId": "XXXXXXXX",
    "lifeTimeEnergy": 1875.9724827777784
  }
]
```

Just need to fetch this and match the charger ID to get the correct value. This is a synchronous API, so it will give us exact results. And it solves the race condition problem, too.

Reply:
> I just tested this API call here: https://developer.easee.cloud/reference/get_api-sites-siteid-energy

I'll have a look into this. But I think/fear this could maybe deliver just an up to an hour old value. As I see it, the poll API polls an actual reading from the box. We have to check "your" API above while charging too see If the value changes with every call.
For the poll -> Productupdate it Changes. Maybe thats the cause for the 3 Minute Limit.

will check and process your other comments tomorrow...

Reply:
Thanks for going the extra mile and checking the behavior of the API.

So this would leave us with 
- a poll request after the op_mode changes (to capture data as accurate as possible, especially on disconnect). This is what the current state of the PR provides.
- additionally we would want to poll this regularly, independent of any op_mode changes

Is my understanding correct? If so, will you put in the poll? It would involve using a [ticker](https://gobyexample.com/tickers), and letting it run in a go [routine](https://gobyexample.com/goroutines) launched while we init the charger. This might sound "scary" at first, but given your code here so far, I am pretty sure you can do it. 💪

Reply:
Yes, that's correct so far.
You mean I can do it.. let's see. BTW, my IDE is vi ;)
I updated the PR, could you please review the ticker + routine stuff, and the updates done to ChargedEnergy and if the new func REquestLifetimeEnergyUpdate is ok.. thanks!